pkg/webhook/tenant/forbidden_annotations_regex.go

```diff
@@ -0,0 +1,76 @@
+// SPDX-License-Identifier: Apache-2.0
+
+//nolint:dupl
```

Comment:
I guess there's no issue with `dupl` linter here, let's try to remove and run locally the linters to check if this is really required.